mne/viz/ica.py

```diff
@@ -787,6 +787,8 @@ def _plot_sources_raw(ica, raw, picks, exclude, start, stop, show, title,
     inds = list(range(len(picks)))
     data = np.array(data)
     n_channels = min([20, len(picks)])
+    raw._first_time = raw.first_samp / float(raw.info['sfreq'])
```

Comment:
I wasn't suggesting monkey-patching, no. A `@property` would be the cleanest way to do it:
```
@property
def _first_time(self):
    return self._first_samp / float(self.info['sfreq'])
```